### PR TITLE
Change/email enroled users

### PR DESCRIPTION
**Pull request by gkaragia**
#### Changes proposed in this Pull Request:
With this change, an emailed is sent to enroled users only when a lesson is dripped.

#### Testing instructions:

* Have a course with a lesson which drips today.
* Have a few users that are enrolled to the course and a few which have progress but are not currently enrolled.
* Trigger the cronjob which sends an email to the users (`woo_scd_daily_cron_hook`)
* Observe that an email is sent to the users that are enroled only.

### CODE REVIEW

File: includes/class-sensei-content-drip.php [hunk 1]
```diff
@@ -415,4 +415,17 @@ public function load_template( $name, $args = array() ) {
 		$theme_path   = 'sensei-content-drip/';
 		Sensei_Templates::get_template( $name, $args, $theme_path, $default_path );
 	}
+
+	/**
+	 * Check if course enrolment is handled by the legacy method.
+	 *
+	 * @since 2.0.2
+	 */
+	public function is_legancy_enrolment() {
```
Comment (renatho) [inline includes/class-sensei-content-drip.php]: Typo on the method name "lega**n**cy".

**Comment by jom** [comment]
It looks like `\Scd_Ext_Utils::get_course_users` has one other usage (which is fine to be in another PR). Did you consider putting this legacy check logic in that method? Seems fine here, too, just wondering what we might end up doing with the original method.

**Comment by gkaragia** [comment]
> Did you consider putting this legacy check logic in that method?

Yes this was my initial approach as it fits better there, but I thought that I will deprecate the method. Then I saw that it is also used elsewhere where we want the learners with progress and didn't deprecate it but the solution stuck 😄 I'll refactor.

File: includes/class-sensei-content-drip.php (hunk 1 above)
Comment (gkaragia) [reply includes/class-sensei-content-drip.php]: Fixed in ea32108b0257905a88d5e6f24e760f16fded879c

**Comment by gkaragia** [comment]
Hey @jom I am merging since this PR has a requirement for my current task. Let me know if you have any suggestions `get_course_users` and I'll handle it in a new one.

**Pull request merged**